drivers/sensor/mcux_tempmon/Kconfig

```diff
@@ -0,0 +1,12 @@
+# NXP MCUX temperature monitor (TEMPMON)
+
+# Copyright (c) 2023 Centralp
+# SPDX-License-Identifier: Apache-2.0
+
+config MCUX_TEMPMON
+	bool "NXP MCUX temperature monitor (TEMPMON)"
+	default y
+	depends on DT_HAS_NXP_MCUX_TEMPMON_ENABLED
```

Comment:
this should be `DT_HAS_NXP_TEMPMON_ENABLED` since you renamed it to `nxp,tempmon`

Reply:
Yes, fixed.